assignment1/src/main/java/ro/utcn/sd/he/assignment1/persistence/jdbc/JdbcQuestionTagRepository.java

```diff
@@ -0,0 +1,55 @@
+package ro.utcn.sd.he.assignment1.persistence.jdbc;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import ro.utcn.sd.he.assignment1.model.Question;
+import ro.utcn.sd.he.assignment1.model.Tag;
+import ro.utcn.sd.he.assignment1.persistence.api.QuestionTagRepository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RequiredArgsConstructor
+public class JdbcQuestionTagRepository implements QuestionTagRepository {
+    private final JdbcTemplate template;
+
+    @Override
+    public List<Question> getQuestionsWithTag(Tag tag) {
+        return template.query("Select * FROM question JOIN question_tag ON question.id = question_tag.questionID" +
+                "                              JOIN tag ON question_tag.tagID = tag.id WHERE tag.id = ?",
+                (resultSet, i) -> new Question(
+                        resultSet.getInt("id"),
+                        resultSet.getString("author"),
+                        resultSet.getString("title"),
+                        resultSet.getString("text"),
+                        resultSet.getTimestamp("creation_date_time")
+                ),
+                tag.getId());
+    }
+
+    @Override
+    public List<Tag> getTagsOfQuestion(Question question) {
```

Comment:
Use your IDE to reformat your code, e.g. https://www.jetbrains.com/help/idea/using-code-editor.html#reformat_rearrange_code. This will add spaces and wrap the code appropriately.